src/libraries/Common/tests/System/IO/Compression/ZipTestHelper.cs

```diff
@@ -333,6 +333,20 @@ public static async Task DirsEqual(string actual, string expected)
             await ItemEqual(actualList, expectedList, isFile: true);
             await ItemEqual(actualFolders, expectedList, isFile: false);
         }
+        public static void NormalizeFileNames(string folder)
```

Comment:
Instead o normalizing the names on the disk, I would fix the comparison in the function below, to take normalization into account, and do that only for the affected platform.